types/eris-sharder/index.d.ts

```diff
@@ -0,0 +1,118 @@
+// Type definitions for Eris-Sharder 1.10
+// Project: https://github.com/discordware/eris-sharder
+// Definitions by: MathleteDev <https://github.com/mathletedev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as Eris from 'eris';
+import { EventEmitter } from 'events';
+
+export class Cluster {
```

Comment:
Where does this entity comes from? It's not listed in the exports IMO:
https://github.com/discordware/eris-sharder/blob/dev/src/index.js#L4-L5

Reply:
Oh sorry, I forgot to remove that export. I'll fix that.

Reply:
@mathletedev can you re-evaluate the outcome? 
TL;DR;
it's ok to `export interface Cluster` I think

I've re-read the source code, and I believe, your intent was correct. in the 'sendTo' client is expected to send cluster param, so at least the shape of the cluser should be exported. But instead of exporting class (which is not an option in native JS), one can export type (Interface), so consuming code could create an object represeting cluster. Sorry, I'm not an user of the package, just read the source code, thx!

Reply:
Could you take a look at it? I changed the class to an interface and exported it. (Sorry, I'm still new to this and this is the first pull request I've done)